WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewAllFragment.kt

```diff
@@ -89,35 +84,52 @@ class StatsViewAllFragment : DaggerFragment() {
         super.onSaveInstanceState(outState)
     }
 
-    private fun initializeViews(savedInstanceState: Bundle?) {
+    private fun StatsViewAllFragmentBinding.initializeViews(savedInstanceState: Bundle?) {
         val layoutManager = LinearLayoutManager(activity, RecyclerView.VERTICAL, false)
 
         savedInstanceState?.getParcelable<Parcelable>(listStateKey)?.let {
             layoutManager.onRestoreInstanceState(it)
         }
-
-        recyclerView.layoutManager = layoutManager
-        loadingRecyclerView.layoutManager = LinearLayoutManager(activity, RecyclerView.VERTICAL, false)
+        with(statsListFragment) {
+            recyclerView.layoutManager = layoutManager
+            loadingRecyclerView.layoutManager = LinearLayoutManager(activity, RecyclerView.VERTICAL, false)
+        }
 
         swipeToRefreshHelper = WPSwipeToRefreshHelper.buildSwipeToRefreshHelper(pullToRefresh) {
             viewModel.onPullToRefresh()
         }
 
-        nextDateButton.setOnClickListener {
-            viewModel.onNextDateSelected()
-        }
-        previousDateButton.setOnClickListener {
-            viewModel.onPreviousDateSelected()
+        with(statsListFragment.dateSelector) {
+            nextDateButton.setOnClickListener {
+                viewModel.onNextDateSelected()
+            }
+            previousDateButton.setOnClickListener {
+                viewModel.onPreviousDateSelected()
+            }
         }
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
         val nonNullActivity = requireActivity()
+        with(StatsViewAllFragmentBinding.bind(requireView())) {
```

Comment:
Sorry, I missed this one the first time!

```suggestion
        with(StatsViewAllFragmentBinding.bind(view)) {
```